CHANGELOG.md

```diff
@@ -3,15 +3,23 @@
 ## 3.0
 
 ### NEXT
-* performance optimization: Instantiate fewer KmmResults
 
+* Fix ASN.1 decoding flaw for a very specific length encoding
+* Performance optimization: Instantiate fewer KmmResults
 * Move `PemEncodable`/`PemDecodable` from _indispensable_ to _indispensable-asn1_ module.
+* More comprehensive PEM encoding/decoding support:
+    * `CryptoPublicKey`
+      * Note that PKCS1 encoding of RSA keys is not supported as it is discouraged (decoding is supported)
+      * ANSI encoding and decoding is also unsupported, because decoding requires context and encoding this way is incomplete
```

Comment:
can you give me some details on this?

Reply:
you won't have curve infos, etc.

Reply:
> you won't have curve infos, etc.

In the encoded data.
I maintain it is fine that we can decode and encode into the go-to format

Reply:
> is this not what `.WithoutCurve` already expresses?

I don't recall the details, but it is not feasible in the PEM code path. you'd need curve guessing and I prefer not to guess during decoding

Reply:
no, but you could have `CryptoPublicKey.FromANSI` which is a `PemDecodable` to `CryptoPublicKey.EC.WithoutCurve` or whatever

Reply:
see #238